steps/apps-decompiler/1.0.0/step.yml

```diff
@@ -0,0 +1,79 @@
+title: Mobile apps decompiler
+summary: |
+  Decompile mobile APK and/or IPA to look inside
+description: |
+  Decompile mobile APK and/or IPA to look inside
+
+  ### Troubleshooting
+
+  - Be aware, you have to launch this step after the compilation step ("Android Build" for android, "Xcode Archive" for iOS)
+  - Launch "Deploy to Bitrise.io" step after this one to deploy report files to artifacts
+website: https://github.com/odemolliens/bitrise-step-apps-decompiler
+source_code_url: https://github.com/odemolliens/bitrise-step-apps-decompiler
+support_url: https://github.com/odemolliens/bitrise-step-apps-decompiler/issues
+published_at: 2021-01-21T12:04:43.593758+01:00
+source:
+  git: https://github.com/odemolliens/bitrise-step-apps-decompiler.git
+  commit: 9d55d0cb5e332c4be62952c78329efbf8dd5f8e8
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
```

Comment:
What about other multi-platform ones?

Reply:
Based on 
https://github.com/bitrise-steplib/steps-xcode-archive/blob/master/step.yml
and
https://github.com/bitrise-steplib/steps-gradle-runner/blob/master/step.yml

which generate the APK and IPA
I don't test it with the others so I don't know if it can be used, so to be sure, I didn't add them.